MatomoTracker.php

```diff
@@ -1949,17 +1949,19 @@ protected function sendRequest($url, $method = 'GET', $data = null, $force = fal
             }
         }
 
+        $content = '';
+
         if (function_exists('curl_init') && function_exists('curl_exec')) {
             $options = $this->prepareCurlOptions($url, $method, $data, $forcePostUrlEncoded);
 
             $ch = curl_init();
             curl_setopt_array($ch, $options);
             ob_start();
             $response = @curl_exec($ch);
+            curl_close($ch);
```

Comment:
This might need to be done at a later step. `curl_error` use below might always hold an empty result if the connection is closed before calling it.

Reply:
@sgiehl oh, yes, you are right. Changed position.